### PR TITLE
automl_video_classification_model_evaluation

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Model evaluation component added
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1391 @@
+{
```
Comment (ivanmkc) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Copyright 2021 Google LLC
</pre><p>2022</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ivanmkc) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    IMPORT_FILE = "gs://automl-video-demo-data/hmdb_split1_5classes_train_inf.csv"
</pre><p>Needs to be in the right bucket. See notebook workflow doc for details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>Combine these 2 cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>Install the packages required for executing this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>Put the google-cloud-aiplatform, google-cloud-pipeline-components and kfp on a single pip3 install -- to resolve dependencies</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>Combine 2 text cells.</p><p>Remove 'Skip this step.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>when you deployed the model -&gt; when you trained the model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>Prepair -&gt; Prepare</p><p>Test -&gt; test</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>Video batch can only be in JSONL format</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

[the remark this answers is not in the document]
Comment (udaypunna) [reply notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <p>written in List model evaluation from training section.</p><p>if not please give me more details.</p><p> </p>

**Review by andrewferlitsch — APPROVED** [review]

File: notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb [hunk 2]
```diff
@@ -0,0 +1,1358 @@
...
```
Comment (KevinBNaughton) [inline notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #90.            dataflow_use_public_ips=dataflow_use_public_ips,
</pre><p>Do we want to shows users these fields in this notebook? From lines 85 to 90. They are more advanced functionality &amp; I don't see explanations around these fields. Maybe a callout to the docs page about additional dataflow configuration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

**Pull request merged**